bin/create_table.py

```diff
@@ -10,59 +10,98 @@ def parse_args(args=None):
     Description = "Create a table by parsing json output to extract N50, BUSCO, QV and COMPLETENESS stats."
 
     parser = argparse.ArgumentParser(description=Description)
-    parser.add_argument("DATATYPE", help="Data type and kmer size for MerquryFK.")
-    parser.add_argument("N50", help="Input GOAT N50 JSON file.")
-    parser.add_argument("BUSCO", help="Input BUSCO short summary JSON file.")
-    parser.add_argument("QV", help="Input QV TSV file from MERQURYFK.")
-    parser.add_argument("COMPLETENESS", help="Input COMPLETENESS stats TSV file from MERQURYFK.")
-    parser.add_argument("FILE_OUT", help="Output CSV file.")
-    parser.add_argument('--version', action='version', version='%(prog)s 1.0')
+    parser.add_argument("-g", "--genome", help="Input NCBI genome summary JSON file.")
+    parser.add_argument("-s", "--sequence", help="Input NCBI sequence summary JSON file.")
+    parser.add_argument("-b", "--busco", help="Input BUSCO short summary JSON file.")
+    parser.add_argument("-p", "--pacbio", help="PacBio sample ID used for MerquryFK.")
+    parser.add_argument("-q", "--qv", help="Input QV TSV file from MERQURYFK.")
+    parser.add_argument("-c", "--completeness", help="Input COMPLETENESS stats TSV file from MERQURYFK.")
+    parser.add_argument("-o", "--outcsv", help="Output CSV file.")
+    parser.add_argument("-v", "--version", action="version", version="%(prog)s 1.0")
```

Comment:
Major version bump since the way the parameters are given changed.
```suggestion
    parser.add_argument("-v", "--version", action="version", version="%(prog)s 2.0")
```